### PR TITLE
Misc improvements

### PR DESCRIPTION
**Pull request by Moetto**

### CODE REVIEW

File: buildSrc/build.gradle.kts [hunk 1]
```diff
@@ -9,7 +9,9 @@ repositories {
 }
 
 dependencies {
+    implementation("org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.30-RC")
     implementation("org.jetbrains.kotlin:kotlin-gradle-plugin")
```
Comment (Glukoosi) [inline buildSrc/build.gradle.kts:13]: Do we need both?

File: buildSrc/src/main/kotlin/vauhtijuoksu-api.kotlin-common-conventions.gradle.kts [hunk 2]
```diff
@@ -28,7 +30,7 @@ dependencies {
     implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
 
     // Use JUnit Jupiter API for testing.
-    testImplementation("org.junit.jupiter:junit-jupiter-api:5.6.2")
+    testImplementation("org.junit.jupiter:junit-jupiter-api")
```
Comment (Glukoosi) [inline buildSrc/src/main/kotlin/vauhtijuoksu-api.kotlin-common-conventions.gradle.kts:33]: Why the version is removed here but not at line 23?

File: .gitignore [hunk 3]
```diff
@@ -5,3 +5,5 @@
 build
 
 .idea
+
+python
```
Comment (Glukoosi) [inline .gitignore:9]: just out of curiosity when we are getting this kind of file(?)

Comment (Moetto) [reply .gitignore:9]: I use it for azure client

File: buildSrc/build.gradle.kts (hunk 1 above)
Comment (Moetto) [reply buildSrc/build.gradle.kts:13]: No, I'll remove the latter

File: buildSrc/src/main/kotlin/vauhtijuoksu-api.kotlin-common-conventions.gradle.kts (hunk 2 above)
Comment (Moetto) [reply buildSrc/src/main/kotlin/vauhtijuoksu-api.kotlin-common-conventions.gradle.kts:33]: It's in the constraints block, which sets the version for all the submodules we might end up using.

**Review by Glukoosi — APPROVED** [review]

**Comment by Moetto** [comment]
Cleaned up the review commit, needs re-approval pls @Glukoosi

**Review by Glukoosi — APPROVED** [review]

**Pull request merged**